src/rpcmining.cpp

```diff
@@ -429,6 +436,18 @@ Value getblocktemplate(const Array& params, bool fHelp)
         }
         else
             throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid mode");
+
+        const Value& capsval = find_value(oparam, "capabilities");
+        if (capsval.type() == array_type) {
+            const Array& caps = capsval.get_array();
+            for (size_t i = 0; i < caps.size(); i++) {
+                auto cap = caps[i].get_str();
+                if (cap == "coinbaseval") {
```

Comment:
https://github.com/bitcoin/bips/blob/master/bip-0022.mediawiki#block-template-request mentions ``coinbasevalue`` but not ``coinbaseval`` -- is the latter correct?